src/include/miopen/db_record.hpp

```diff
@@ -223,8 +223,8 @@ class DbRecord
 
         const bool ok = values.Deserialize(s);
         if(!ok)
-            MIOPEN_LOG_WE(
-                "Perf db record is obsolete or corrupt: " << s << ". Performance may degrade.");
+            MIOPEN_LOG_I("Perf db record is obsolete or corrupt: " << s
```

Comment:
If you follow [this recommendation](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1214#issuecomment-938204105), then please use `MIOPEN_LOG_IE` (info message for release builds, error for DEV or debug builds).